.eslintrc.js

```diff
@@ -5,5 +5,6 @@ module.exports = {
     '@typescript-eslint/array-type': ['error', { default: 'generic' }], // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/array-type.md
     '@typescript-eslint/consistent-type-imports': 'error', // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/consistent-type-imports.md
     'react/no-unescaped-entities': 'off', // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md
+    'react/react-in-jsx-scope': 'off',
```

Comment:
I suggest adding this link to the page that explains this rule (as a nice to have) `// https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md`

Reply:
Alright, let me add it.